scripts/php.sh

```diff
@@ -56,6 +60,7 @@ function run() {
 	cp -f /usr/share/doc/$VERSION-*/php.ini-development /etc/php.ini
 	sed -i "s/;date\.timezone.*/date\.timezone = ${TZ}/g" /etc/php.ini
 	sed -i "s/memory_limit.*/memory_limit = ${MEMORY_LIMIT}M/g" /etc/php.ini
+	sed -i "s/max_execution_time.*/max_execution_time = ${EXEC_TIME}/g" /etc/php.ini
```

Comment:
I don't think so.

1. It sets it to 60 seconds, a sensible default for a vagrant box IMO
2. The dev can change this via the `-e` flag (why warn them of something they did on purpose)
3. If we warn on this then why not every setting
4. The warnings are just going to get lost in a ton of output so I doubt anyone will notice it anyway